_posts/2016-03-10-glimpse-of-the-naive-bayes-classifier.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: post
+comments: true
+
+title: Glimpse Of The Naive Bayes' Classifier
+excerpt: A simple machine learning algorithm explained
+author: Riddhish Bhalodia
+category: [Machine Learning]
+tags: [ML,OpenCV]
+
+assets_dir: /assets/glimpse-of-the-naive-bayes-classifier
+---
+
+### <u>Glimpse Of The Naive Bayes' Classifier</u>
+
+![](../assets/img/joke.png)
```

Comment:
Use `{{ page.assets_dir }}` instead of `../assets/img/` here and two other places.